docs/guide/architecture-styles/event-driven-content.md

```diff
@@ -24,6 +24,14 @@ The source of the events may be external to the system, such as physical devices
 
 In the logical diagram above, each type of consumer is shown as a single box. In practice, it's common to have multiple instances of a consumer, to avoid having the consumer become a single point of failure in system. Multiple instances might also be necessary to handle the volume and frequency of events. Also, a single consumer might process events on multiple threads. This can create challenges if events must be processed in order or require exactly-once semantics. See [Minimize Coordination][minimize-coordination].
 
+There are two primary topologies within the event-driven architecture:
+
+- **Broker topology**. Components broadcast occurrences as events to the entire system, and other components either act upon the event or just ignore the event. This topology is useful when event processing flow is relatively simple. There is no central coordination or orchestration, so this topology can be very dynamic. This topology is highly decoupled which helps provide scalability, responsiveness, and component fault tolerance. No component owns or is aware of the state of any multi-step business transaction and actions are taken asynchronously. Subsequently, distributed transactions are risky because there is no native means to be restarted or replayed. Error handling and manual intervention strategies needs to be carefully considered as this topology can be a source of data inconsistency.
+
+- **Mediator topology**. This topology addresses some of the shortcomings of broker topology. There is an event mediator that manages and controls the flow of events. The event mediator maintains the state, manages error handling and restart capabilities. Unlike broker topology, components broadcast occurrences as commands and only to designated channels, usually message queues. These commands are not expected to be ignored by their consumers. This topology offers more control, better distributed error handling, and potentially better data consistency. This topology does introduce increased coupling between components and the event mediator could become a bottleneck or a reliability concern.
+
+If you need error handling capability and control over flow of events, you should use mediator topology. But if you need higher performance and higher scalability, you should choose broker topology. You can also use a hybrid model combining both mediator and broker topologies.
+
```

Comment:
We haven't even told them "when to use this architecture" yet -- vs even deciding on which approach to use.  As I suggested before, lets leave this out.  Especially since the new topology text doesn't even have "performance" in there yet it's here in the "choose" text.

```suggestion
```